docs/index.md

```diff
@@ -78,6 +78,15 @@ works and how Okio does it.
 Okio’s implementation language from Java to Kotlin.
 
 
+Requirements
+------------
+
+Okio supports Android 4.0.3+ (API level 15+) and Java 7+.
```

Comment:
Previously our minimums weren’t specified! Note that this is lower than OkHttp’s minimums.

Reply:
I’m assuming 4.0.3 is the lowest version that anyone is targeting for anything.  Should we go lower?

Reply:
No, I don't think so